files/sunrpc_tcp_settings.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eu
+
+# Load the sunrpc module and apply the sysctls configured for the subsystem.
+# This will be run from a startup service to ensure the tunables are always set
+# on boot.
+modprobe sunrpc
```

Comment:
Although this will work, the usual way to ensure kernel modules are loaded at boot time is described here - https://www.baeldung.com/linux/kernel-module-load-boot

Reply:
That's the Linux way, I'll apply this